app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params.dig(:session, :email)&.downcase
+    if user&.authenticate params[:session][:password]
+      log_in user
+      redirect_to user_url(user)
+    else
+      flash[:danger] = t "login.invalid_email_password_combination"
```

Comment:
```suggestion
      flash.now[:danger] = t "login.invalid_email_password_combination"
```
check và test lại code nhé, chỗ này phải .now nữa